cmake/dependencies.cmake

```diff
@@ -1,32 +1,33 @@
 find_package(PkgConfig REQUIRED)
 include(FetchContent)
 
-################################################################################
+# ##############################################################################
```

Comment:
It looks like somehow a lot of formatting changes ended up in this file. Could you checkout this file from the main branch?

Reply:
This was because I added the Google Benchmark library here (although I'm not certain if this is the correct method for adding a dependency). I removed it because benchmarking and comparing my code to LAPACK will be addressed in a new issue and pull request.